db/event_manager_test.go

```diff
@@ -333,6 +337,26 @@ func (wr *WebhookRequest) Clear() {
 	wr.mutex.Unlock()
 }
 
+func (em *EventManager) waitForProcessedTotal(ctx context.Context, waitCount int, maxWaitTime time.Duration) error {
+	startTime := time.Now()
+
+	worker := func() (bool, error, interface{}) {
+		eventTotal := em.GetEventsProcessedSuccess() + em.GetEventsProcessedFail()
+		if eventTotal == int64(waitCount) {
```

Comment:
This seems like it might be more intuitive and reliable for callers as eventTotal >= int64(waitCount)